src/SamplesApp/SamplesApp.UITests/Helpers/UITestHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using SamplesApp.UITests;
+
+namespace UITests.Shared.Helpers
+{
+	public static class UITestHelper
+	{
+		public static bool CompareScreenShots(FileInfo img1, FileInfo img2, float thresholdPercentage = 1f) => CompareScreenShots(
+			new Bitmap(img1.FullName),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'img1' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826516)